src/containers/sidebar/sidebar.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+
+import Drawer from '@material-ui/core/Drawer';
+import ListItemText from '@material-ui/core/ListItemText';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import Divider from '@material-ui/core/Divider';
+import { useStyles } from './sidebar.styles';
+
+import { getCategoryURL } from '../../pages/home/categories-list/categories-list';
+
+const Sidebar = ({ setMenuOpen, menu }) => {
+  const styles = useStyles();
+  const { categories, language } = useSelector(({ Categories, Language }) => ({
+    categories: Categories.list,
+    language: Language.language
+  }));
+
+  const menuList = categories.map(({ _id, name, isMain }) =>
```

Comment:
It is better firstly to filter only categories that have `isMain: true` and then map it to the components array

Reply:
changed